db/redis.go

```diff
@@ -5,25 +5,37 @@ package db
 import (
 	"bytes"
 	"encoding/gob"
-	"strconv"
-	"time"
-
 	"github.com/anonyindian/logger"
 	"github.com/gigauserbot/giga/config"
 	"github.com/go-redis/redis"
+	"log"
```

Comment:
Please **DO NOT** use log package, we've our custom logger, `github.com/anonyindian/logger`, use that.